new/prompts.js

```diff
@@ -1,3 +1,15 @@
+const chalk = require('chalk')
+
+const defaultOrCustom = {
+  type: "list",
+  message: "Please select a preset: ",
+  name: "preset",
+  choices: [
+    { name: `default: (${chalk.yellow`React`}, ${chalk.yellow`React-Router`}, ${chalk.yellow`Redux`}, ${chalk.yellow`Express`}, ${chalk.yellow`MongoDB`}, ${chalk.yellow`Mongoose`})`, value: "react-default" },
```

Comment:
I was thinking, it might make more sense to leave Redux out of the default config. Rationale: 
1. the React community is trying to move away from Redux in favor of smaller/simpler options like React Context, except in edge cases. 
2. Users who need redux can just add redux after install, correct? 

If so, I would just have the default be React w/ React-Router.